### PR TITLE
SSCS-1869 session time out

### PR DESCRIPTION
**Pull request by bendiggle**
- Add meta tag to head of globals.html which redirects the user to `/session-timeout` after 20 minutes which ends the session.
- Add session-timeout page.
- Add SESSION_TIME_OUT env var and set default timeout to 20 minutes.
- Add unit tests

### CODE REVIEW

File: steps.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable max-len */
 const { concat } = require('lodash');
 const Entry = require('steps/entry/Entry');
-const Exit = require('steps/exit/Exit');
+const Exit = require('steps/exit/exit/Exit');
```
Comment (barbaracassani) [inline steps.js]: exit/exit/Exit :)

File: config/default.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "redis": {
     "url": "redis://127.0.0.1:6379",
     "secret": "defaultsecret",
-    "useSSL": "false"
+    "useSSL": "false",
+    "timeout": 1200
```
Comment (barbaracassani) [inline config/default.json]: 1200 in seconds it's 20 mins but the copy says 30

Comment (paulgain) [reply config/default.json]: We need to add a line of content to the bottom of each step warning the user of the 20 minute timeout. As it stands the timeout is very abrupt and without warning. 
"This page will time out in 20 minutes if there is no activity. This is to protect your data."

Going forward https://tools.hmcts.net/jira/browse/SSCS-3180 will address the issue by posing a question "Want more time to continue" with yay/nay buttons.

Comment (bendiggle) [reply config/default.json]: @paulgain Okie dokie! Will add that later.

**Pull request merged**